app/templates/buyers/brief_responses.html

```diff
@@ -39,6 +39,58 @@
     <div class="column-two-thirds">
         <div class="dmspeak">
           {% if response_counts['eligible'] > 0 %}
+            {% if ods %}
+            <p>
+              <span class='visual-emphasis'>{{ response_counts['eligible'] }} {{ pluralize(response_counts['eligible'], "supplier", "suppliers") }}</span>
+              responded to your requirements and {{ pluralize(response_counts['eligible'], "meets", "meet") }} all your essential skills and experience. Suppliers that didn't meet all your essential requirements have already been told they were unsuccessful.
+            </p>
+            <p>
+              You said you’d take <span class='visual-emphasis'>{{ brief.numberOfSuppliers }} {{ pluralize(brief.numberOfSuppliers, "supplier", "suppliers") }}</span> through to the evaluation stage. To do this, you need to:
+            </p>
+            <ol class="instruction-list">
```

Comment:
Is there a reason for using the raw HTML for the instruction list pattern rather than the jinja macro?

I'd be better to use the latter if possible.